### PR TITLE
Radio Button

### PR DESCRIPTION
**Pull request by vNakamura**
affects: `@crave/farmblocks-input-radio`

ISSUES CLOSED: #280

### CODE REVIEW

File: packages/input-radio/src/styledComponents/Label.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import styled, { css } from "styled-components";
+import { colors } from "@crave/farmblocks-theme";
+
+const enabledtyles = props =>
```
Comment (fgameiro) [inline packages/input-radio/src/styledComponents/Label.js]: Typo? `enabledStyles`?

Comment (vNakamura) [reply packages/input-radio/src/styledComponents/Label.js]: yep, thanks!

File: packages/input-radio/src/Radio.test.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+      component.setProps({ checked: true });
+      expect(component.state().checked).toBe(true);
+    });
+    test("should not update state unnecessarily", () => {
+      const component = shallow(<Radio checked />);
+      component.instance().setState = setStateSpy;
+      component.setProps({ checked: true });
+      component.setProps({ unrelatedProp: false });
+      expect(setStateSpy).toHaveBeenCalledTimes(0);
```
Comment (alcferreira) [inline packages/input-radio/src/Radio.test.js]: Good 👍

File: packages/input-radio/src/RadioGroup.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      <Group>
+        {React.Children.map(children, child => {
+          const checked = child.props.value === value;
+          const childProps = {
+            checked,
+            disabled: child.props.disabled || disabled
+          };
+          return React.cloneElement(child, {
+            ...allChildrenProps,
+            ...childProps
```
Comment (alcferreira) [inline packages/input-radio/src/RadioGroup.js]: ✅

**Pull request merged**